examples/tonic-key-value-store/src/main.rs

```diff
@@ -1,3 +1,8 @@
+fn main() {
+    eprint!("this example has not yet been updated to hyper 1.0");
```

Comment:
Why not? I did it with little effort for the `tower-async` version. Want me to contribute this as a PR to your PR?

Reply:
How did you port it? Tonic doesn't support http 1.0. Did you do a bunch of mapping of the types?

Reply:
Sorry I was wrong. Tonic and Warp examples I deleted during my original port as I do not use these libraries. I only kept in the hyper and axum ones and those are the only once I fixed this time as well.